src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -512,3 +512,10 @@ def labels(self) -> List[str]:
     @handle_orchestrator_error('host')
     def inventory(self, refresh=None):
         return get_inventories(None, refresh)
+
+    @Endpoint('GET')
+    @ReadPermission
+    @raise_if_no_orchestrator([OrchFeature.HOST_LIST])
+    @handle_orchestrator_error('host')
+    def all_hosts(self):
```

Comment:
and also add a description about this method on why its needed.
```suggestion
    def list(self):
```